pubpub/pubpub
# add header with collapsible navigation

## Issue(s) Resolved
#2597

## Test Plan

1. Go to the `eric` community.
2. Verify the header and footer are acceptable recreations of the reference implementation.

## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

### Supporting Docs
